RecoMuon/Configuration/python/RecoMuon_EventContent_cff.py

```diff
@@ -36,6 +36,15 @@
                                            #'drop *_muons_muons1stStep2muonsMap_*',
                                            #'drop recoIsoDepositedmValueMap_muons_*_*', #not really used
                                            #'drop doubleedmValueMap_muons_muPFIso*_*', #already inside the muon
+                                           # displacedMuons collection
+                                           'keep recoMuons_displacedMuons_*_*',
```

Comment:
Do you know what is the contribution to the total size increase given by each of these new collections?

Reply:
We have computed it in a ttbar sample (runTheMatrix.py workflow 11634.0) and we have a table in slide 28 of this presentation: https://cernbox.cern.ch/index.php/s/FfAZonfSest7kqQ